apps/wallets/smart-wallet/next/turbo.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://turbo.build/schema.json",
+    "extends": ["//"],
+    "tasks": {
+        "build": {
+            "dependsOn": ["^build"],
+            "inputs": [".env", ".env.local"],
```

Comment:
should these applications also use tsup or not worth it? (not for this PR)

Reply:
i dont even think its possible to use tsup for a nextjs/create react app